pages/events/summer-school.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <main class="summer-school-page">
+    <AppPageHeaderWithCard
+      :description="eventData.header.description"
+      :cta="eventData.header.cta"
+      :card-title="eventData.header.cardSectionHeading"
+    >
+      <template slot="title">
+        {{ eventData.header.primaryTitle }}
+        <br>
+        {{ eventData.header.secondaryTitle }}
+      </template>
+      <template slot="card">
+        <EventCard v-bind="eventData.header.card" vertical-layout>
+          {{ eventData.header.card.description }}
+        </EventCard>
+      </template>
+    </AppPageHeaderWithCard>
+
+    <AppMosaicSection
+      class="summer-school-page__section"
+      :title="eventData.mosaic.title"
+      :elements="eventData.mosaic.elements"
+    />
+
+    <section class="summer-school-page__section">
+      <h2 class="copy__title">
+        {{ eventData.agenda.title }}
+      </h2>
+      <client-only>
```

Comment:
I just tested removing the `client-only` wrapper and it seems to work fine. Can we remove it from here or is there a reason why we need it?

Reply:
Removed